src/modules/JiraApis.psm1

```diff
@@ -0,0 +1,358 @@
+$JIRA_HELP_URL = "https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-issueidorkey-transitions-post"
+
+function Invoke-HandleBadRequest {
+    Param (
+        [hashtable]$content,
+        [PSCustomObject]$Issue
+    )
+  
+    If ($content.errorMessages.Count -eq 0) {
+        $content.errorMessages = "Error on Transition. See $( $env:GITHUB_ACTION_URL ?? $JIRA_HELP_URL ) for help."
+    }
+  
+    $fieldIdToName = @{}
+    ForEach ($fieldId in $content.errors.Keys) {
+      $field = $Issue.editmeta.fields.$fieldId
+      If ($null -ne $field -And $field.name -ne $fieldId -And $content.errors[$fieldId] -inotlike "*$( $field.name )*") {
```

Comment:
Sometimes a bad request doesn't give you enough information to determine what field it really is speaking to.  Added a `names` property that expands on the field name.

![image](https://github.com/im-open/transition-jira-tasks-by-query/assets/8923247/84648775-3853-43b7-ae49-02d015dbe91d)